content/onboarding/learners/meet-the-team/_index.md

```diff
@@ -3,4 +3,34 @@ _db_id: 881
 content_type: topic
 ready: true
 title: Meet the Umuzi team
----
\ No newline at end of file
+---
+
+![staffprofiless](staff_profiless.png)
+
+#**Welcome to the family!**
+
+
+Below are a few key people you should know as well as what they are all about. 
+Some you will interact with and others not too much but they will all contribute to your learning journey here at Umuzi in one way or another.
+
+
+![Andrew](Andrew.png)
```

Comment:
I would probably remove email addresses from the pics?

Reply:
I've corrected this :)